calico_containers/calico_ctl/status.py

```diff
@@ -35,6 +36,13 @@ def status(arguments):
     this file's docstring with docopt
     :return: None
     """
+    try:
+        docker_client.version()
```

Comment:
There's a lot of docker calls throughout `status` which gather info. Wrapping the whole thing in a try/except means that If one of the initial calls passes fine but the later ones do not, the "Docker is not running" message wouldn't accurately capture the real problem.